content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,744 @@
+    kubectl apply -f - --context=${CTX_CLUSTER2}
+{{< /text >}}
+
+Install a remote secret in `CLUSTER1` that provides access to `CLUSTER2`’s API server.
+
+{{< text bash >}}
+$ istioctl x create-remote-secret \
+    --context=${CTX_CLUSTER2} \
+    --name=CLUSTER2 | \
+    kubectl apply -f - --context=${CTX_CLUSTER1}
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="Multi-Primary, Multi-Network" category-value="multi-primary-multi-network" >}}
+
+The following steps will install the Istio control plane on both `CLUSTER1` and
+`CLUSTER2`, making each a {{< gloss >}}primary cluster{{< /gloss >}}. `CLUSTER1` is
+on the `NETWORK1` network, while `CLUSTER2` is on the `NETWORK2` network. This means there
+is no direct connectivity between pods across cluster boundaries.
+
+Both `CLUSTER1` and `CLUSTER2` observe the API Servers in each cluster for endpoints.
+
+Service workloads across cluster boundaries communicate indirectly, via
+dedicated gateways for [east-west](https://en.wikipedia.org/wiki/East-west_traffic)
+traffic. The gateway in each cluster must be reachable from the other cluster.
+
+{{< image width="75%"
+    link="multi-primary-multi-network.svg"
+    caption="Multiple primary clusters on separate networks"
+    >}}
+
+- **Configure `CLUSTER1` as a primary with services exposed**
+
+Install Istio on `CLUSTER1`:
+
+{{< text bash >}}
+$ cat <<EOF > ./cluster1.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: MESH1
+      multiCluster:
+        clusterName: CLUSTER1
+      network: NETWORK1
+      meshNetworks:
+        NETWORK1:
+          endpoints:
+          - fromRegistry: CLUSTER1
+          gateways:
+          - registryServiceName: istio-eastwestgateway.istio-system.svc.cluster.local
+            port: 15443
+        NETWORK2:
+          endpoints:
+          - fromRegistry: CLUSTER2
+          gateways:
+          - registryServiceName: istio-eastwestgateway.istio-system.svc.cluster.local
+            port: 15443
+EOF
+$ istioctl install --context=${CTX_CLUSTER1} -f cluster1.yaml
+{{< /text >}}
+
+Install a gateway in `CLUSTER1` that is dedicated to
+[east-west](https://en.wikipedia.org/wiki/East-west_traffic) traffic. By
+default, this gateway will be public on the Internet. Production systems may
+require additional access restrictions (e.g. via firewall rules) to prevent
+external attacks. Check with your cloud vendor to see what options are
+available.
+
+{{< text bash >}}
+$ CLUSTER=CLUSTER1 NETWORK=NETWORK1 \
+    samples/multicluster/gen-eastwest-gateway.sh | \
+    kubectl apply --context=${CTX_CLUSTER1} -f -
+{{< /text >}}
+
+Since the clusters are on separate networks, we need to expose all services
+(*.local) on the east-west gateway in both clusters. While this gateway is
+public on the Internet, services behind it can only be accessed by services
+with a trusted mTLS certificate and workload ID, just as if they were on the
+same network.
```

Comment:
Ok this is where I think there is some gap when comparing with our previous replicated control plane model.   The doc here instructs users to expose all services while the replicated control plane instructs users to selectively expose services.   I don't think we have that function ready yet as we are waiting on k8s for the multicluster service API.

This is also where I'm not sure if we are ready to remove the replicated control plane doc yet without an equivalent replacement.  

@nmittler @stevenctl @rshriram @frankbu

Reply:
I'm not sure there is a gap, really. This should be the default behavior for a service mesh that spans multiple clusters. The updated [Deployment Models](https://preliminary.istio.io/latest/docs/ops/deployment/deployment-models/#dns-with-multiple-clusters) doc discusses selective service exposure via `Service`/`ServiceEntry`, but that is beyond the scope of this guide.

Reply:
Are you saying it is exposed to the other cluster by default but what services to import are controlled by creating service entry or the corresponding service in the consuming cluster?

Reply:
You can further clamp it down by being more selective when we expose services on the gateway. I can add a note in the text if that helps?